server/controllers/templates/web_templates.go

```diff
@@ -110,15 +110,34 @@ var IndexTemplate = template.Must(template.New("index.html.tmpl").Parse(`
     <p class="title-heading small"><strong>Locks</strong></p>
     {{ if .Locks }}
     {{ $basePath := .CleanedBasePath }}
+    <div class="lock-grid">
+    <div class="lock-header">
+      <span>Repository</span>
+      <span>Project</span>
+      <span>Workspace</span>
+      <span>Date/Time</span>
```

Comment:
How come date/time are switching with status?

Reply:
They are not swapped visually on the grid. The previous implementation used a strange mix of float left and right that caused the properties to be switched.